File: Solutions/867. Transpose Matrix/867.java
class Solution {
    public int[][] transpose(int[][] m) {
        int[][] ans = new int[m[0].length][m.length];
        for(int i=0;i<m.length;i++){
            for(int j=0;j<m[0].length;j++){
                ans[j][i] = m[i][j];
            }
        }
        return ans;
    }
}

File: Solutions/867. Transpose Matrix/867.py
class Solution:
    def transpose(self, m: List[List[int]]) -> List[List[int]]:
        ans = [[None]*len(m) for _ in range(len(m[0]))]
        for i in range(len(m)):
            for j in range(len(m[0])):
                ans[j][i] = m[i][j]
        return ans 